### ISSUE TITLE
[FEATURE] Add Role Commands

### ISSUE THREAD

**Issue by ghost** [anf/BotANF#9]
Like ,role [ROLENAME]

**Comment by anf** [comment]
That's what we do not have the intention of doing right now. However, we'll surely add it as soon as the "base" of the project gets built i.e. the desired architecture, the modules, the database connection. We're even thinking of redesigning BotANF v2 into a native ASP .Net Core 3.1 application, just because of app secrets and other plans. All that is to be done and as soon as we implement all that, we'll get started on the commands. Feel free to leave this issue open.

TL;DR: We will do this later.

### What are we up to right now?
We are trying to discuss in the [discord server](https://discord.gg/fKWpK7A) about the plans. Feel free to join in and contribute to the project :wink:
